src/server_manager/install_scripts/install_server.sh

```diff
@@ -378,14 +378,18 @@ install_shadowbox() {
   readonly SB_IMAGE=${SB_IMAGE:-quay.io/outline/shadowbox:stable}
 
   log_for_sentry "Setting PUBLIC_HOSTNAME"
-  # TODO(fortuna): Make sure this is IPv4
```

Comment:
We still need to ensure it's IPv4. If we get IPv6 it won't work, so I'd keep the comment there.

Reply:
It seems we can pass `--ipv4` to curl to ensure that.

Reply:
I've switched to long-form arguments everywhere so it's clear what we're doing.